runtime/lua/vim/filetype.lua

```diff
@@ -635,6 +639,12 @@ local extension = {
   tssop = "tssop",
   tutor = "tutor",
   twig = "twig",
+  ts = function(name, bufnr)
+    if getline(bufnr, 1):find("<%?xml") then
+      return "xml"
+    else
+      return "typescript"
+  end,
```

Comment:
I think you're missing an `end` here
```suggestion
    else
      return "typescript"
    end
  end,
```